build-tools/packages/build-tools/src/type-test-configure/main.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import program from "commander";
+
+import { updatePackageJsonFile, updateTypeTestConfiguration } from "./typeTestConfigUtil";
+
+program
+	.option("-r|--reset", "Reset broken types data")
+	.option("-p|--previous <version string>", "set the version to test against")
+	.parse(process.argv);
+
+updatePackageJsonFile(".", (json) =>
+	updateTypeTestConfiguration(json, {
+		resetBroken: !!program.reset,
+		version: program.previous,
+	}),
+);
```

Comment:
Did you want to leave this?

Reply:
I was thinking about implementing a flub package command for "exec", then it could call this instead of needing a dedicated flub command, and we could at least compare the two approaches. Given how fast configure runs though, I think using the generate script for that would make a better comparison, so I'll remove this.